### PR TITLE
📖 Spanish docs fixes

### PR DESCRIPTION
**Pull request by pablopunk**
* Spelling mistakes
* Rephrasing
* Broken links

### CODE REVIEW

**Comment by feross** [comment]
Hey @pablopunk, thanks for the PR! While I do know a bit of Spanish, I'm not an expert by any means. @falmar – what are your thoughts on this PR? If it looks good to you, then feel free to hit the merge button. 🚢

File: docs/README-esla.md [hunk 1]
```diff
@@ -75,35 +75,35 @@ que usan `standard`!
...
-- Inicio Rapido
+- Inicio Rápido
   - [Instalación](#instalación)
   - [Uso](#uso)
-  - [Lo que podrias hacer si eres inteligente](#lo-que-prodias-hacer-si-eres-inteligente)
+  - [Lo que podrías hacer si eres inteligente](#lo-que-podrías-hacer-si-eres-inteligente)
 - FAQ
-  - [¿Porque deberia usar JavaScript Standard Style?](#porque-deberia-usar-javascript-standard-style)
-  - [¿Quien usa JavaScript Standard Style?](#quien-usa-javascript-standard-style)
+  - [¿Por qué debería usar JavaScript Standard Style?](#por-qué-deberia-usar-javascript-standard-style)
```
Comment (feross) [inline docs/README-esla.md]: Are you sure that the accents will appear in the anchor link here?

**Comment by falmar** [comment]
The spelling and rephrase is fine. Actually I couldn't use the accents myself since I switched to US keyboard :/... The links seems to work just for me. I checked them [here](https://github.com/pablopunk/standard/blob/f737f2299d2cfd0ba7d3787c53df081398cec66f/docs/README-esla.md).

@pablopunk The only issue I see are the links which are off the list formatting

**Comment by feross** [comment]
@pablopunk Do you know how to update a PR? You can read about how to do that here: http://stackoverflow.com/a/9790530

Once you have fixed @falmar's feedback, this PR can be merged. Thanks!

**Comment by pablopunk** [comment]
@falmar oh, don't know what happened there. It should be fixed now.

**Comment by feross** [comment]
Test failure is not the fault of this PR. I fixed the cause, restarting the tests...

**Comment by feross** [comment]
We're all green, so merging! 🚢 🚢 🚢

**Pull request merged**